app/settings/datasources/datasources.controller.js

```diff
@@ -88,7 +88,7 @@ function (
         }
     };
 
-    $q.all([DataProviderEndpoint.queryFresh().$promise, ConfigEndpoint.get({ id: 'data-provider' }).$promise]).then(function (response) {
+    $q.all([DataProviderEndpoint.queryFresh().$promise, ConfigEndpoint.getFresh({ id: 'data-provider' }).$promise]).then(function (response) {
```

Comment:
Same as above, but less dramatically since we should really not cache data providers.